### PR TITLE
Old Ahelp

### PR DESCRIPTION
**Pull request by ghost**
# Description

bring back old ahelp

:cl:
- tweak: revert to old ahelp

### CODE REVIEW

File: Content.Client/UserInterface/Systems/Bwoink/AHelpUIController.cs [hunk 1]
```diff
@@ -46,9 +46,9 @@ public sealed class AHelpUIController: UIController, IOnSystemChanged<BwoinkSyst
     private bool _discordRelayActive;
     private bool _hasUnreadAHelp;
 
-    public const string AHelpErrorSound = "/Audio/Admin/ahelp_error.ogg";
-    public const string AHelpReceiveSound = "/Audio/Admin/ahelp_receive.ogg";
-    public const string AHelpSendSound = "/Audio/Admin/ahelp_send.ogg";
+    public const string AHelpErrorSound = "/Audio/Admin/adminhelp_old.ogg";
+    public const string AHelpReceiveSound = "/Audio/Admin/adminhelp_old.ogg";
+    public const string AHelpSendSound = "/Audio/Admin/adminhelp_old.ogg";
```
Comment (FoxxoTrystan) [inline Content.Client/UserInterface/Systems/Bwoink/AHelpUIController.cs:51]:
```suggestion
    public const string AHelpErrorSound = "/Audio/Admin/adminhelp_old.ogg"; // Floofstation
    public const string AHelpReceiveSound = "/Audio/Admin/adminhelp_old.ogg"; // Floofstation
    public const string AHelpSendSound = "/Audio/Admin/adminhelp_old.ogg"; // Floofstation
```

**Review by ShadesMars — CHANGES_REQUESTED** [review]

File: RobustToolbox
Comment (FoxxoTrystan) [inline RobustToolbox:1]: Why is this here?

**Review by ShadesMars — CHANGES_REQUESTED** [review]

**Review by ShadesMars — APPROVED** [review]
